misc/working_w_l10n_agency.md

```diff
@@ -0,0 +1,109 @@
+
+* Pontoon: set up a “translator” account when there is no community; or a basic “contributor” account when a community exists so that the vendor provides suggestions and the community review them.
+* Bugzilla: so that the vendor’s project manager can interact in bugs. Bugzilla will allow us to have a central place to post project details and updates. Communication is generally handled via email with the vendor, but it’s useful to have a bug number to reference to, and the vendor can use the bug to quickly access all the details.
+* Google Drive: create a shared folder for the vendor to view, review, and translate directly spreadsheets or Google Docs.
+* GitHub: give commit permission if they need to commit directly in a repository (e.g. [Legal docs](https://github.com/mozilla/legal-docs/))
+
+Note: Additional account setup may be needed if the project does not support our localization process (e.g. Wordpress).
+
+Once accounts are set up, allow the agency to explore the tools, flows and the product features to familiarize itself before actual work starts. Make sure to provide links to documentation as needed on how to use Pontoon and explain how to perform queries on Transvision. Setting up a meeting to share your screen might be a good idea, and use this opportunity to answer their questions about the project or the tools.
+
+## Localizing a project
+
+### Planning the request
+
+Once outsourcing is a necessity for a project, before reaching out to the l10n agency to make an official request, make sure to check the following:
+
+* Required locales.
+* Scope of work: is the request asking for anything other than translation, review and testing?
+* Milestones and Deadline: negotiate it so the agency has sufficient time to finish each of the required tasks, and the community has enough time to review and sign off.
+* Check with communities if they want to fully handle the request, handle only some parts (e.g. initial translation or QA), or can’t work on it at all and let the agency do the work.
+
+### Quotation
+
+Now that we have all the initial info we need, we are good to go ahead and reach out to the vendor and provide what they need to generate a quote.
+
+* Get a quote from the agency by communicating volume, locales, deadline, level of service needed, and type of content.
+
+### Setting everything up for the request
+
+Now that we got a quote, we need to get it approved, send content to the vendor, and check permission.
+
+* Provide quote/estimate to the project requestor for approval.
+* For all the target locales, make sure both agency and the community have the proper access to the l10n tool or platform where the localizable content is staged and roles properly assigned per agreement.
+[//]: <> (* Share the scope, the plan, and the expectations to both the agency and the community.)
+[//]: <> (* Communicate the plan and the finalized schedule with the project stakeholders.)
```

Comment:
@peiying2 I’ve put those two lines in comment (well, tried but syntax is wrong) because I’m not sure I fully understand them. Could you help clarify what you have in mind for `plan`, `scope` and `expectations`? Or provide examples of what those are